Fix inability of configure script to detect that it is running on macOS
because it uses $CPP to do so but CPP hadn't been set to anything yet.
https://salsa.debian.org/clint/fakeroot/-/merge_requests/16
--- configure.ac.orig	2022-05-22 12:14:31.000000000 -0500
+++ configure.ac	2022-05-23 02:20:22.000000000 -0500
@@ -7,6 +7,7 @@
 AM_INIT_AUTOMAKE
 AM_MAINTAINER_MODE
 AC_CONFIG_HEADERS([config.h])
+AC_PROG_CPP
 AC_PROG_MAKE_SET
 LT_INIT
 LT_LANG(C)
